doc/netplan-yaml.md

```diff
@@ -1317,6 +1322,31 @@ The specific settings for bridges are defined below.
     > used.
 
 
+## Properties for device type `dummies:`
+
+**Status**: Optional.
+
+**Purpose**: Use the `dummies` key to create virtual interfaces.
+
+**Structure**: The key consists of a mapping of interface names.
+Dummies are virtual devices that can be used to route packets to
+without actually transmitting them.
+
+```yaml
+network:
+  dummies:
+    dm0:
+      addresses:
+        - 192.168.0.123/24
+      ...
+```
+
+When applied, a virtual interface called `dm0` will be created in the system.
+
+[See this section](#properties-for-all-device-types) for the list of properties that can be
+used with this type of interface.
```

Comment:
How does this look inside the rendered manpage? We don't really have hyperlinks/anchors there.

```suggestion
See the ["Properties for all device types"](#properties-for-all-device-types) section for the list of properties that can be
used with this type of interface.
```

Reply:
Oops, good point!